### DISCUSSION TITLE
Code Fold Indicators - Code fold line overlay

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#3999] [Ideas]
In another application I tried in the past that contained similar code folding lines/symbols and it had a setting that added the ability to turn on a different color line that was also bold/thick line over the existing code fold lines/outline of symbols.  This line would show up based on what line the cursor was on giving user a graphical indicator to see the current code they were in.
So for example if you were in something like an XML file it would show a bold different color line on top of all the code fold lines connected the code that the cursor was in.
This was a very helpful graphical indicator and I was wondering if something like this would be considered to be added to Geany.

Thank You

**Comment by elextr** [comment]
That is not an existing feature, so "somebody" will have to design and program it.
